rna/rna-bam-sort-and-index/rna-index.inputs.json

```diff
@@ -1,16 +1,16 @@
 {
   "##_COMMENT1": "Set your per-sample values: name and BAM path",
   "RNAIndex.sample_name": "__INSERT_SAMPLE_NAME__",
```

Comment:
The input variable names here should all be changed from RNAIndex to RNABAMSortAndIndex